Makefile

```diff
@@ -381,6 +381,7 @@ partialclean::
 	rm -f compilerlibs/ocamlopttoplevel.cmxa
 
 ocamlnat: compilerlibs/ocamlcommon.cmxa compilerlibs/ocamloptcomp.cmxa \
+    compilerlibs/ocamlbytecomp.cmxa \
```

Comment:
Same question as above.  And also (if ocamlbytecomp is indeed required), you should udate Makefile.nt as well.